server/tabby_server/api/base64_to_image.py

```diff
@@ -0,0 +1,41 @@
+import base64
+import cv2
+import io
+import numpy as np
+
+
+""" Example Useage
+    import tabby_server.api.base64_to_image as base64_to_image
+    from PIL import Image
```

Comment:
You can actually see this line should solve the issue if it was not commented. Just be sure to add pillow to the poetry list.